backend/src/services/userOrganizationService.ts

```diff
@@ -77,3 +88,36 @@ export const getUserCurOrganization = async (
   const response: IUserOrganization = { role, name };
   return response;
 };
+
+export const switchUserToOrganization = async (
+  organizationId: string,
+  user: ITransportedUser
+): Promise<ITransportedUser> => {
+  const organization = await getCustomRepository(OrganizationRepository).getById(organizationId);
+  if (!organization) {
+    throw new CustomError('Organization not found', 404);
+  }
+  const userOrganizationExist = await getCustomRepository(UserOrganizationRepository)
+    .getUserOrganization(organizationId, user.id);
+  if (userOrganizationExist) {
+    throw new CustomError('User already switch to this organization', 400);
+  }
+  await getCustomRepository(UserOrganizationRepository)
+    .addUserOrganization(user.id, {
+      role: Role.DEVELOPER,
+      userId: user.id,
+      organizationId,
+      status: OrganizationStatus.ACTIVE
+    });
```

Comment:
Why do we add a user to an organization here? We should have been added the user to the organization when we sent the invite, and during this /switch request we only need to change his `currentOrganizationId`